backend/src/main/scala/akka/viz/events/EventPublisherActor.scala

```diff
@@ -10,8 +10,9 @@ class EventPublisherActor extends Actor with ActorLogging {
   var subscribers = immutable.Set[ActorRef]()
   var availableTypes = immutable.Set[Class[_ <: Any]]()
   var eventCounter = 0L
+  var replayables: Vector[ReplayableEvent] = Vector.empty
```

Comment:
And I guess it might be a good idea to just build single ActorState with all actorrefs, their classnames and if they are alive or not